script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -1025,7 +1025,7 @@ If I do not burn it to the ground, it would go against everything that I believe
 [sha:1c569a09df5b70346eb15174ccaa0b9e0ed30421]{
 -- Page 870, Offset 7977.
 -- 真祖の姫。その心臓を、おれに<呉|よ>こせ”」
-Princess of the True Ancestors. That heart of yours, shall be mine."
+%{i}Princess of the True Ancestors%{/i}. That heart of yours, shall be mine."
```

Comment:
Not sure I'd italicize this one either. It just doesn't feel like a nickname to me, only a description